mdoc-sbt/src/sbt-test/sbt-mdoc/scalajs-1.7/project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.7.0")
```

Comment:
Frozen tests at this Scala.js version as it's the last one that was working in the old setup of the plugin, so we want to keep it here to make sure future changes won't break older versions.